index.Rmd

```diff
@@ -136,9 +136,9 @@ p1 <- crime_income %>%
 This figure is a map of crime by type in 2017
 
 ```{r}
-p1 + geom_point(crime_income, aes(x = crime_date, y = Assault))
+p1 + geom_point(crime_income, aes(x = crime_date, y = Assault))  
 
-ggplot(crime_income, aes(x = crime_date, y = Assault)) + geom_point()
+ggplot(crime_income, aes(x = crime_date, y = Assault)) + geom_point()   
```

Comment:
This one yielded an error, Error: `mapping` must be created by `aes()` . You clearly have an aes call there, so I'm not sure if this is pointing to an issue with p1, or just user error on my end.